docs/src/componentDocs/CollapsibleHeaderLayout/examples/CollapsibleHeaderLayoutExample.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import {ExampleShowcase} from '../../../shared';
+import {Header} from '@brightlayer-ui/react-native-components';
+import image from '../images/farm.jpg';
+
+export const CollapsibleHeaderLayoutExample = (): JSX.Element => (
+  <ExampleShowcase>
+    <Header
```

Comment:
As this example is for `CollapsibleHeaderLayout`, we should import and display that instead of `Header` like https://github.com/etn-ccis/blui-react-native-showcase-demo/blob/97656a1a8d7d76a4b898cd9eb8cbe550ee83d1cd/App.tsx#L37